### PR TITLE
Update of the FRENCH translation for the XBMC VIDEO SERVER tool

### PR DESCRIPTION
**Pull request by Galixte**
Hi @Jalle19,

I have updated the FRENCH translation for your tool.

Could you merge this, please?

Regards.

### CODE REVIEW

File: src/protected/messages/fr/Settings.php [hunk 1]
```diff
@@ -32,9 +27,12 @@
   'Before you get started, please have a look at the application settings' => 'Avant toute utilisation de cette application, il est important de s’intéresser à ses paramètres.',
   'Cache all API results' => 'Mettre en cache tous les résultats de l’API',
   'Changing this will reset the default language for all users' => 'Ce changement aura pour effet d’assigner la langue par défaut à tous les utilisateurs. Chaque utilisateur pourra à son tour modifier la langue.',
-  'Determines how long the application should wait for a response from XBMC. Increase this if you get timeout errors.' => 'Détermine le temps d’attente de l’application pour obtenir une réponse de Kodi. Augmenter cette valeur si le message suivant apparait : « Le temps d’attente de la page a expiré. ».',
```
Comment (Jalle19) [inline src/protected/messages/fr/Settings.php]: Please don't move the lines, it makes it harder to re-generate these files when new strings are added

File: src/protected/messages/fr/Movies.php [hunk 2]
```diff
@@ -18,7 +18,7 @@
  */
 return array (
   'Click an image to see other movies with that person, or click the name to go to the person\'s IMDb page' => 'Cliquer sur l’image d’un(e) acteur(trice) pour voir tous ses films disponibles. Cliquer sur son nom pour accéder à la page de sa filmographie sur le site Web IMDb.com.',
-  'Director: {director}' => 'Réalisateur(trice) : {director}',
+  'Director: {director}' => 'Réalisateur(trice) : {director}',
```
Comment (Jalle19) [inline src/protected/messages/fr/Movies.php]: I don't know what editor you're using but for some reason some of these spaces have changed?

File: src/protected/messages/fr/Settings.php (hunk 1 above)
Comment (Jalle19) [reply src/protected/messages/fr/Settings.php]: Actually I guess this is okay

File: src/protected/messages/fr/Movies.php (hunk 2 above)
Comment (Jalle19) [reply src/protected/messages/fr/Movies.php]: Probably the file encoding is wrong? Your German changes were fine.

Comment (Galixte) [reply src/protected/messages/fr/Movies.php]: My FRENCH too, in French we use a non-breaking space before ```:```, ```?```, ```!```, ```;```, ```›```, ```»``` and after ```«``` and ```‹```. Source: https://en.wikipedia.org/wiki/Non-breaking_space#Width_variation.

That’s why I added the missing non-breaking spaces.

Comment (Jalle19) [reply src/protected/messages/fr/Movies.php]: Okay, cool

File: src/protected/messages/fr/Settings.php (hunk 1 above)
Comment (Galixte) [reply src/protected/messages/fr/Settings.php]: @Jalle19, oh I’m sorry I didn’t see this comment. Do I do anything, a new PR or something else?

**Pull request merged**